### DISCUSSION TITLE
(multidimensional) Cloud Raster Format

### DISCUSSION THREAD

**Discussion by pinkerltm** [cloudnativegeo/cloud-optimized-geospatial-formats-guide#116] [General]
Recently I stumbled over a "native Cloud Raster Format" of a large geospatial software vendor ;-) which should be able to store any multidimensional data in a cloud optimized way.

Anybody has any information about that? Is it something completly new or is it a derivative of NetCDF/HDF ? Will GDAL be able to write or even read that? Can't find anything about it except Help/Support-Articles about its usage.

**Comment by vincentsarago** [comment]
https://zarr.readthedocs.io/en/stable/getting_started.html

**Comment by j08lue** [comment]
> Anybody has any information about that?

Are you looking for technical details about how [ESRI Cloud Raster Format (CRF)](https://pro.arcgis.com/en/pro-app/latest/help/data/imagery/an-overview-of-multidimensional-raster-data.htm) is implemented?

I don't have any - it's proprietary and poorly documented, especially compared to open formats like Zarr.

But if you dare to call out "You-Know-Who" by their real name, Google spits out a bit of details in [this presentation](https://www.esri.com/content/dam/esrisites/en-us/about/events/media/UC-2019/technical-workshops/tw-5755-977.pdf)

![image](https://github.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/assets/3404817/f81637de-5be1-4f48-9a31-d604bde1bf47)

And maybe the answers you are looking for are buried in this magic 7000 line Python file? 😉 https://github.com/Esri/OptimizeRasters/blob/master/OptimizeRasters.py (I doubt it, though)

**Reply by pinkerltm** [reply]
Thanks! Thats great background info, exactly what I was looking for. I just was curious about this as I am quite aware about Zarr and didn't know there is already something compareable in the commercial sector, so this hit me unexpected. After knowing what it does and how it works I feel more confident to tell people why Zarr might be the better choice for their valuable data assets ;-)
